ChangeLog

```diff
@@ -1,3 +1,11 @@
+== 2018-04-17 version 0.16.1
+
+
+
+== 2018-04-14 version 0.16.0
+
+* bump version#205
```

Comment:
Need more information what is changed in this version..
https://github.com/treasure-data/td/compare/v0.15.9...v0.16.0

Reply:
Added more description!